packages/web/src/utils/helpers.tsx

```diff
@@ -1,3 +1,9 @@
+import React from 'react';
+
 export function formatNumber(x: number) {
   return x.toLocaleString();
 }
+
+export function repeat(times: number, children: React.ReactNode) {
+  return [...Array(times)].map((_, idx) => <React.Fragment key={idx}>{children}</React.Fragment>);
```

Comment:
Please refrain from calling constructors without a `new` keyword. Also, spread can be replaced with `.fill` as in 
`new Array(times).fill(undefined).map(...)`.